menyer/ラーメン/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,300&display=swap">
+    <link rel="stylesheet" href="style.css">
+    <title>ラーメン</title>
+</head>
+
+<body>
+    <header>
+        <article>
```

Comment:
I would not put an article inside a header. I think you can remove it and style the header instead.